rootfs/api/middleware.py

```diff
@@ -46,5 +46,7 @@ def process_response(self, request, response):
         a response header.
         """
         # clients shouldn't care about the patch release
-        response['DEIS_API_VERSION'] = __version__.rsplit('.', 1)[0]
+        version = __version__.rsplit('.', 1)[0]
+        response['DEIS_API_VERSION'] = version
+        response['DEIS_PLATFORM_VERSION'] = version
```

Comment:
It could be used to scan for vulnerable deis installs, but it also can be useful for checking if the controller has a required bugfix. If you think it's a large concern, we could only expose the header to authenticated requests.